### ISSUE TITLE
varcheck doesn't support vendoring

### ISSUE THREAD

**Issue by stevenh** [opennota/check#32]
If you try to use varcheck on code which has vendored dependencies it reports bogus errors

For example:
```
main.go:6:2:warning: unused global variable could not import github.com/pkg/errors (cannot find package "github.com/pkg/errors" in any of: (varcheck)
main.go:11:14:warning: unused global variable undeclared name: errors (varcheck)
```
Where main.go clearly imports errors:
```go
package main

import (
        "fmt"

        "github.com/pkg/errors"
)

func main() {
        err := fmt.Errorf("some error")
        fmt.Println(errors.Wrap(err, "my wrapped error"))
}
```

To ensure you get this error you need github.com/pkg/errors vendored and it must NOT be in your GOPATH.

Also reported to gometalinter:  https://github.com/alecthomas/gometalinter/issues/190

**Comment by stevenh** [comment]
fixed now
